ci/packaging/suse/rpmfiles_maker.sh

```diff
@@ -41,6 +41,10 @@ then
       "$branch_project" "$branch_name"
     osc -A 'https://api.suse.de' co -o "$work_dir" \
       "$branch_project/$branch_name"
+    log "Running OBS services"
+    pushd "$work_dir"
+    osc -A 'https://api.suse.de' service disabledrun
```

Comment:
`osc disabledrun` is listed in the docs as deprecated (yes, I had to look it up :D).  We probably shouldn't be developing new code using deprecated calls.  Wouldn't this have the same effect?
```suggestion
    osc -A 'https://api.suse.de' service run go_modules
```
Or perhaps `runall`.

Reply:
Good catch, I had no idea. Looks like with `disabledrun` set in the `_service` file in https://build.suse.de/request/show/213484 that should work fine.